resources/lang/fr/buttons.php

```diff
@@ -18,14 +18,14 @@
             'users' => [
                 'activate'           => 'Activer',
                 'change_password'    => 'Changer de mot de passe',
-                'clear_session'         => 'Clear Session',
-                'confirm'             => 'Confirm',
+                'clear_session'       => 'Effacer la session',
```

Comment:
Alignment with other => I mean this line has different alignment than line 22, 21